example/glue/ObjectModel.hpp

```diff
@@ -570,95 +313,23 @@ class GC_ObjectModel
 	MMINLINE void
 	fixupForwardedObject(MM_ForwardedHeader *forwardedHeader, omrobjectptr_t destinationObjectPtr, uintptr_t objectAge)
```

Comment:
The code that is common _is_ in the base class: 
```
GC_ObjectModelBase::fixupForwardedObject(forwardedHeader, destinationObjectPtr, objectAge);
```
is implementation for this function. In j9 GC_ObjectModel, base class method is called before implementing j9-specific fixup.